api/ethnicities/ethnicityModel.js

```diff
@@ -0,0 +1,32 @@
+const knex = require('../../data/db-config');
+
+const findAll = async () => {
+  return await knex('ethnicities');
+};
+
+const findById = async (id) => {
+  return await knex('ethnicities').where('ethnicity_id', id).first();
+  // keeping these for now as a reference
+  // .select(knex.raw('ethnicities.*, to_json(locations.*) as location'))
```

Comment:
I see that this code has been commented out. Can we please remove it as per our standards?